src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    protected String color;
```

Comment:
Consider making 'color' private and providing a protected setter if needed, to better encapsulate the property.